pom.xml

```diff
@@ -155,12 +156,6 @@ THE SOFTWARE.
       <optional>true</optional><!-- no need to have this at runtime -->
     </dependency>
     <dependency>
-      <groupId>com.github.stephenc.findbugs</groupId>
```

Comment:
@stephenc I think it's no longer required, right? Or is it a licensing issue again?